packages/components/src/Description/styles.ts

```diff
@@ -2,11 +2,13 @@ import { styled } from '../styles';
 
 export const Wrapper = styled.div<{
   justifyContent?: 'space-between' | 'start';
+  leader?: boolean;
 }>`
   display: flex;
-  flex-wrap: wrap;
   align-items: baseline;
   justify-content: ${({ justifyContent }) => justifyContent};
+
+  text-align: ${({ leader }) => (leader ? 'right' : 'left')};
```

Comment:
```suggestion
  text-align: ${({ $leader }) => ($leader ? 'right' : 'left')};
```

Все пропсы, которые не должны попадать в dom помечаем символом $